doc/source/tutorials/local/Exploring_data/Searching_and_plotting.rst

```diff
@@ -85,11 +85,33 @@ to extract, for example, just the methane data:
     tac_surface_search = search_surface(site="TAC", species="ch4")
     tac_surface_search.results
 
+Keyword options when searching
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When searching it is also possible to specify multiple options for keywords.
+If this is done using a list, then datasources which have any of the
+specified values will be found. For example if we wanted to search for methane
+at two specific inlets we could write:
+
+.. code:: ipython3
+
+    from openghg.retrieve import search_surface
+
+    tac_surface_search = search_surface(site="TAC", species="ch4", inlet=["100m", "185m"])
+    tac_surface_search.results    
+
+This will return results from both the 100m and 185m inlets (but not the 54m inlet).
+
+Note: it is also possible to specify a dictionary to provide an option between 
```

Comment:
Do we need to mention an option that isn't covered by the tutorial?

Reply:
Is there a downside to mentioning it? Mentioning it exists at least means if someone has an alternative use for it that wasn't previously considered they know it is there as an option.

Reply:
Yeah that's cool.